Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocLargeEventCollectionViewCell.swift

```diff
@@ -270,12 +289,26 @@ class ArticleAsLivingDocLargeEventCollectionViewCell: CollectionViewCell {
         case .thankAndViewChanges:
             contentView.addSubview(thankButton)
             contentView.addSubview(viewChangesButton)
+            
+            thankButton.titleLabel?.font = UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+            viewChangesButton.titleLabel?.font = UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+
+            if largeEvent.wereThanksSent {
+                thankButton.setImage(UIImage(named: "thank")?.withRenderingMode(.alwaysTemplate), for: .normal)
+                thankButton.setTitle(WMFLocalizedString("aaald-events-thanked-title", value: "Thanked", comment: "Button title after a user thanks an editor - past tense of 'thank'"), for: .normal)
+            } else {
+                thankButton.setImage(UIImage(named: "thank-unfilled")?.withRenderingMode(.alwaysTemplate), for: .normal)
```

Comment:
Same rendering mode to asset catalog comment.

Reply:
Updated, thanks.